### ISSUE TITLE
Loom: Create testing plan

### ISSUE THREAD

**Issue by tajila** [eclipse-openj9/openj9#15186]
Initially, we can write some small sanity test cases.

Basic test 1:

```
        public static void main(String[] args) throws Throwable {
                Thread t = Thread.ofVirtual().name("duke").unstarted(()->{
                        System.out.println("loom virtual thread");
                });

                t.start();

                t.join();

        }
```

Basic test 2:

```
        public static void main(String[] args) throws Throwable {

                try (var executor = Executors.newVirtualThreadPerTaskExecutor()) {
                        IntStream.range(0, 6).forEach(i -> {
                                executor.submit(() -> {
                                        System.out.println("loom thread " + i);
                                        Thread.sleep(Duration.ofSeconds(1));
                                        System.out.println("loom thread end " + i);
                                        return i;
                                });
                        });
                }

        }
```

It looks like there is a lot of test coverage in `test/java/lang/Thread/virtual/*`

**Comment by tajila** [comment]
FYI @llxia @fengxue-IS @JasonFengJ9

**Comment by fengxue-IS** [comment]
Basic testing merged in #15574

**Comment by fengxue-IS** [comment]
functional testing have been merged and enabled for different JIT/GC/Platform, verbose testing done through sanity.openjdk and extended.openjdk.
closing this issue
